src/app/app.module.ts

```diff
@@ -60,6 +67,7 @@ const materialModules = [
         AboutComponent,
         PatientTableComponent,
         AgePipe,
+        UserInputDialComponent,
```

Comment:
`UserInputDialComponent` -> I think it would be better to name it as `UserInputDialogComponent`

Reply:
done